### PR TITLE
Food

### PR DESCRIPTION
**Pull request by henriklovhaug**
Find recipes. (Only norwegian language supported)

### CODE REVIEW

File: ababot/bot/src/commands/food/recipe_response.rs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        if temp.len() == 3
+            && !temp.get(0).unwrap().is_empty()
+            && !temp.get(1).unwrap().is_empty()
+            && !temp.get(2).unwrap().is_empty()
+        {
+            ingredients.push(Ingredient {
+                amount: temp.get(0).unwrap().to_string(),
+                unit: temp.get(1).unwrap().to_string(),
+                name: temp.get(2).unwrap().to_string(),
+            });
```
Comment (henriklovhaug) [inline ababot/bot/src/commands/food/recipe_response.rs]: Any improvment tips here?

Comment (henriklovhaug) [reply ababot/bot/src/commands/food/recipe_response.rs]: I fixed

File: ababot/bot/src/commands/food/food_task.rs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            m.kind(InteractionResponseType::Modal)
+                .interaction_response_data(|d| {
+                    d.content("Food")
+                        .custom_id("food")
+                        .title("Enter ingredients. På norsk :D")
+                        .components(|c| {
+                            c.create_action_row(|row| {
+                                row.create_input_text(|menu| {
+                                    menu.custom_id("ing1")
+                                        .placeholder("Føøøød")
```
Comment (ghost) [inline ababot/bot/src/commands/food/food_task.rs:24]: Place holder or final?

**Review by mack-stefan — APPROVED** [review]

File: ababot/bot/src/commands/food/food_task.rs (hunk 2 above)
Comment (henriklovhaug) [reply ababot/bot/src/commands/food/food_task.rs:24]: final

**Pull request merged**